HISTORY.md

```diff
@@ -3,8 +3,82 @@
 ## Catalyst unreleased (master branch)
 
 ## Catalyst 14.0
-- The `reactionparams`, `numreactionparams`, and `reactionparamsmap`  functions have been removed.
+
+#### Breaking changes 
+Catalyst v14 was prompted by the release of ModelingToolkit v9. This introduced several breaking changes to the package. A summary of these (and how to handle them) can be found [here](https://docs.sciml.ai/Catalyst/stable/v14_migration_guide/). These are briefly summarised in the following bullet points:
+- `ReactionSystem`s must now be *complete* before they are exposed to most forms of simulation and analysis. With the exception of `ReactionSystem`s created through the `@reaction_network` macro, all `ReactionSystem`s are created incomplete. The `complete` function can be used to generate complete `ReactionSystem`s from incomplete ones.
+- The `states` function has been replaced with `unknowns`. The `get_states` function has been replaced with `get_unknowns`.
+- Support for most units (with the exception of `s`, `m`, `kg`, `A`, `K`, `mol`, and `cd`) has been dropped until further notice.
+- Problem parameter values are now accessed through `prob.ps[p]` (rather than `prob[p]`).
+- A significant bug prevents the safe application of the `remake` function on problems for which `remove_conserved = true` was used.
+- The `reactionparams`, `numreactionparams`, and `reactionparamsmap` functions have been removed.
 - To be more consistent with ModelingToolkit's immutability requirement for systems, we have removed API functions that mutate `ReactionSystem`s such as `addparam!`, `addreaction!`, `addspecies`, `@add_reactions`, and `merge!`. Please use `ModelingToolkit.extend` and `ModelingToolkit.compose` to generate new merged and/or composed `ReactionSystem`s from multiple component systems.
+
+#### General changes
+- The `default_t()` and `default_time_deriv()` functions are now the preferred approaches for creating the default time independent variable and its differential.
+- It is now possible to add metadata to individual reactions, e.g. using:
+```julia
+rn = @reaction_network begin
+    @parameters η
+    k, 2X --> X2, [description="Dimerisation"]
+end
+getdescription(rn)
+```
+a more detailed description can be found [here](https://docs.sciml.ai/Catalyst/dev/model_creation/dsl_advanced/#dsl_advanced_options_reaction_metadata).
```

Comment:
Yes. Implementation of new reaction metadata, and the use of the `hasmetadata` `getmetadata` functions are currently internal and un-documented. If we document these it would be part of the dev-docs on how to create new metadata.

Currently we only support the `noise_scaling`, `description`, and `misc` metatadata, each supported by their own set of getters (MTK style).